### DISCUSSION TITLE
Link title support

### DISCUSSION THREAD

**Discussion by ivankraev** [ueberdosis/tiptap#5013] [General Questions]
### Hi there, I need some help
**I'm trying to set link url, but also link title.
Any suggestions how can I access the text that is associated with the link so I can use it as a title ?
Also how can I set a title along with the link ? I tried**
```
      editor
        .chain()
        .focus()
        .extendMarkRange('link')
        .setLink({ href: url })
        .command(({ tr }) => {
          tr.insertText('your link text')
          return true
        })
        .run()
```
**But it just sets the text and removes the link attributes.**

![Screenshot 2024-03-27 at 17 15 09](https://github.com/ueberdosis/tiptap/assets/89323943/8f3f71a1-27d0-4761-94c9-4a58fe490612)

Any suggestions ?

**Comment by matthiask** [comment]
Here's basically all I had to do to add `title` support to my editor:
https://github.com/matthiask/django-prose-editor/commit/7e0dcfb062787411f832ae5776ae5f90040da45e

Adding the attribute using an `addAttributes` override would probably have been sufficient.
